docs/contributing/01-start-here/05-development.md

```diff
@@ -288,4 +286,4 @@ Adding a new template is straightforward!
 Each template is represented by a folder located at [project-templates](https://github.com/winglang/wing/tree/main/apps/wing/project-templates), containing all of the files that template should be initialized with.
 
 Create a new folder with the template name, and insert any code files that are needed to run it.
-Unit tests ran with `pnpm turbo test` (or in GitHub Actions once you make a pull request) will automatically validate that the template is valid.
+Unit tests ran with `turbo test` (or in GitHub Actions once you make a pull request) will automatically validate that the template is valid.
```

Comment:
Yeah, very handy if you don't install turbo globally. I feel defaulting to `turbo` instead of `pnpm turbo` makes sense (less clutter), and the docs already mentions that you can do both:

> - `pnpm turbo` can be used instead if [Turbo] is not installed globally

So we're good! I'll resolve this comment.